src/CodeStyle/Tools/CodeStyleConfigFileGenerator.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">  
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>net8.0</TargetFramework>
+    <TargetFramework>$(NetCurrent)</TargetFramework>
```

Comment:
We do not allow `$(NetCurrent)` and `$(NetPrevious)` in our project files. Please use `$(SourceBuildToolsetTargetFramework)` instead.

Reply:
That would cause `net6.0` to be used outside of source build: https://github.com/dotnet/roslyn/blob/373a24a97c4e60e9a2b43dcedae59aa724eb7a8a/eng/targets/Settings.props#L96

Reply:
@jaredpar - What would you like me to do here? Should this target `net6.0` or `net8.0`? Should there be a new property defined that can be set to `net8.0` instead of `net6.0` outside the context of source build and if so, what should it be named?